src/api/code_scannings.rs

```diff
@@ -40,4 +41,25 @@ impl<'octo> CodeScanningHandler<'octo> {
     pub fn list(&self) -> list::ListCodeScanningsBuilder<'_, '_> {
         list::ListCodeScanningsBuilder::new(self)
     }
+
+    /// Update an issue in the repository.
```

Comment:
Shouldn't this be updating a code scanning?

Reply:
Good call